README.md

```diff
@@ -28,6 +19,80 @@ git pull upstream main
 14) remove and add our code in frontend/src/app.js
 15) create component folder in frontend/src and create homepage.js in component folder.
 16) Add proxy to frontend/package.json
+------------------------------------------------
+
+## Git Commands
+### Getting & Creating Projects
+| Command | Description |
+| ------- | ----------- |
+| `git init` | Initialize a local Git repository |
+| `git clone ssh://git@github.com/[username]/[repository-name].git` | Create a local copy of a remote repository |
```

Comment:
mention who should use ssh and who should use http while cloning through comment